src/full/Agda/Utils/Update.hs

```diff
@@ -49,7 +51,7 @@ runChangeT :: Functor m => ChangeT m a -> m (a, Bool)
 runChangeT = fmap (mapSnd getAny) . runWriterT . fromChangeT
 
 -- | Run a 'ChangeT' computation, but ignore change flag.
-execChangeT :: Functor m => ChangeT m a -> m a
+execChangeT :: Functor m => ChangeT m a -> m a -- TODO:: Defined but not used
```

Comment:
Please keep, this is a library function.

Reply:
kept